custom/icds/templates/partials/tableau_js.html

```diff
@@ -0,0 +1,213 @@
+<script>
+    var viz = {};
```

Comment:
:+1: to namespace. one other thing that could be useful is pulling out the django template vars into a config object and passing that into your object. that way the javascript can live in its own file (and be tested)